guide/english/c/for/index.md

```diff
@@ -4,7 +4,7 @@ title: For Loop
 
 # For Loop
 
-The `for` loop executes a block of code until the specified condition is false. Use `while` loops when the number of iterations are not known or variable; otherwise, use `for` loops. A common use of `for` loops is array iterations. 
+The `for` loop executes a block of code until the specified condition is false. Use `while` loops when the number of iterations is not known or variable; otherwise, use `for` loops. A common use of `for` loops is array iterations. 
```

Comment:
```suggestion
The `for` loop executes a block of code until the specified condition is false. Use `while` loops when the number of iterations are not known or are variable; otherwise, use `for` loops. A common use of `for` loops is array iterations. 
```
Since iterations is plural, you'll need to use 'are' and not 'is'

Reply:
@thecodingaviator Thank you for your comment. We should use a singular verb 'is'. The verb should be singular because it refers to 'the number'. The subject of this sentence is 'the number'. 'of iterations' is a modifier of 'the number'. \

Please see this link https://english.stackexchange.com/questions/59753/a-number-of-students-vs-the-number-of-students

Reply:
If that seems fine enough to you, leave it so, 

cc: @RandellDawson Does this look fine to you both ways?

Reply:
@thecodingaviator "the number is" is correct grammar in this case.  If the word "a" would have been in front of "number", then it would have been "a number are".  The "the" makes it singular and since the subject is being treated as singular, you must use "the number is".